src/content/en/fundamentals/getting-started/primers/service-workers.md

```diff
@@ -308,12 +308,11 @@ What we are doing is this:
 
 1. Add a callback to `.then()` on the `fetch` request.
 2. Once we get a response, we perform the following checks:
-
-   1. Ensure the response is valid.
-   2. Check the status is `200` on the response.
-   3. Make sure the response type is **basic**, which indicates that it's a 
-      request from our origin. This means that requests to third party assets 
-      aren't cached as well.
+    1. Ensure the response is valid.
```

Comment:
I increased the indentation to render the list items properly on DevSite and GitHub as mentioned in https://github.com/google/WebFundamentals/issues/4422.